doc/supported_file_formats.adoc

```diff
@@ -0,0 +1,50 @@
+= Supported File Formats
+include::../include/config.adoc[]
+:description: The file formats Minetest supports.
+:keywords: fileformats
+
+Minetest is able to read several types of file formats for use as assets in-game, each with varying usecase and support.
+
+== Images
+[grid="all"]
+[options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^"]
+|=======================
+| Format ^| General Usecase | Can be written by Lua | Can be read by Lua | Lossless optimization
+| PNG (`.png`) | All-round good losslessly compressed image format. | Yes (`minetest.encode_png()` ^(5.5+)^) (Third-party library ^(-5.4)^) | Yes (Third-party library) | http://optipng.sourceforge.net/[optipng] (use MTG's https://github.com/minetest/minetest_game/blob/master/utils/optimize_textures.sh[optimize_textures.sh] script for best result)
+| JPEG (`.jpg .jpeg`) | Lossily compressed images (e.g. photorealistic skyboxes and large textures) | No | No .3+| N/A
+| TGA (`.tga`) | Easy to programmatically encode and decode due to being a simple image format | Yes (Third-party library) | No
+| BMP (`.bmp`) | Keeping IrrlichtMt running | No | No
+|=======================
```

Comment:
This is standard AsciiDoc format (and far easier to read/write IMO). Please test before committing this:
```suggestion
== Images
[grid="all"]
[options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^"]
|===
| Format
^| General Usecase
| Can be written by Lua
| Can be read by Lua
| Lossless optimization

| PNG (`.png`)
| All-round good losslessly compressed image format.
| Yes (`minetest.encode_png()` ^(5.5+)^) (Third-party library ^(-5.4)^)
| Yes (Third-party library)
| http://optipng.sourceforge.net/[optipng] (use MTG's https://github.com/minetest/minetest_game/blob/master/utils/optimize_textures.sh[optimize_textures.sh] script for best result)

| JPEG (`.jpg .jpeg`)
| Lossily compressed images (e.g. photorealistic skyboxes and large textures)
| No
| No .3+
| N/A

| TGA (`.tga`)
| Easy to programmatically encode and decode due to being a simple image format
| Yes (Third-party library)
| No

| BMP (`.bmp`)
| Keeping IrrlichtMt running
| No
| No
|===
```

Reply:
done